### PR TITLE
[AC-413] Explain how to view patients and relevant data

### PR DESCRIPTION
**Pull request by f4ww4z**
https://issues.openmrs.org/browse/AC-413

I explained how to search and download patients, and explained how to view their visits, vitals etc.

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -16,19 +16,67 @@
    
 * ### Viewing Patients
 
-   1. Downloading Patients
```
Comment (dkayiwa) [inline features.md:31]: Why did you remove the above header?

File: features.md [hunk 2]
```diff
@@ -16,19 +16,67 @@
...
+   Patients that you have created from the client or synced patients from the server are shown in the **Find Patients** section. From here you can select a patient to do numerous events, such as visiting the patient, capturing vitals and so forth.
 
-   2. Finding Patients
```
Comment (dkayiwa) [inline features.md:33]: Same as above

File: features.md [hunk 3]
```diff
@@ -16,19 +16,67 @@
...
+   #### 1. Downloading Patients
 
-   3. Visits
```
Comment (dkayiwa) [inline features.md:35]: Same as above

File: features.md [hunk 4]
```diff
@@ -16,19 +16,67 @@
...
+   ![Viewing and searching patients](assets/openmrs_android_viewing_patients_1.png)
 
-   4. Diagnosis
```
Comment (dkayiwa) [inline features.md:37]: Same as above

File: features.md [hunk 5]
```diff
@@ -16,19 +16,67 @@
...
+   If this is your first time using the Android client, you are likely to see an empty list of patients in **Find Patients** and you'll want to download and synchronize patients from your server. To do this, click the `+` button from the action bar, then click the search button and enter a patient name. You'll be shown a list of (unsynchronized) patients matching the searched name.
 
-   5. Vitals
```
Comment (dkayiwa) [inline features.md:39]: Same as above

File: features.md [hunk 6]
```diff
@@ -16,19 +16,67 @@
...
+   ![Downloading patients](assets/openmrs_android_viewing_patients_2.png)
 
-   6. Encounters
```
Comment (dkayiwa) [inline features.md:41]: Same as above

File: features.md (hunk 1 above)
Comment (f4ww4z) [reply features.md:31]: Changing this to a slightly bigger header makes it clearer. [See here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R21)

File: features.md (hunk 2 above)
Comment (f4ww4z) [reply features.md:33]: [Likewise](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R31)

File: features.md (hunk 3 above)
Comment (f4ww4z) [reply features.md:35]: [Small header here](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R45)

File: features.md (hunk 4 above)
Comment (f4ww4z) [reply features.md:37]: [Likewise](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R41)

File: features.md (hunk 6 above)
Comment (f4ww4z) [reply features.md:41]: [Changed to a small header](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R53), without numbering

File: features.md (hunk 5 above)
Comment (f4ww4z) [reply features.md:39]: [Changed to a small header](https://github.com/openmrs/openmrs-android-client-user-guide/pull/17/files#diff-e1ec7fcfd5aa4d3ea3e990d9b707a3b4R63), without numbering

**Comment by dkayiwa** [comment]
@shivtej1505 what do you think of this?

**Comment by shivtej1505** [comment]
@dkayiwa, @f4ww4z. This looks perfect. Merging now!

**Comment by f4ww4z** [comment]
Thanks @shivtej1505 !
